mlflow/tracking/fluent.py

```diff
@@ -2244,6 +2244,22 @@ def finalize_logged_model(model_id: str, status: LoggedModelStatus) -> LoggedMod
 
     Returns:
         The updated model.
+
+    Example:
+
+    .. code-block:: python
+        :test:
+
+        import mlflow
+        from mlflow.entities import LoggedModelStatus
+
+        model = mlflow.create_external_model(name="model")
+        loaded_model = mlflow.finalize_logged_model(
```

Comment:
q: shall we say logged_model instead of loaded_model?

Reply:
Makes sense. I'll rename this variable.